### ISSUE TITLE
make install fails on linux

### ISSUE THREAD

**Issue by david9100** [noxdafox/clipspy#45]
`(clipspy) dwood@dw7470u:~/dev/clipspy$ sudo make install
make -f makefile -C clips_source                            \
	CFLAGS="-std=c99 -O3 -fno-strict-aliasing -fPIC"               \
	LDLIBS="-lm -lrt"
make[1]: Entering directory '/home/dwood/dev/clipspy/clips_source'
gcc -o clips main.o -L. -lclips -lm -lrt
make[1]: Leaving directory '/home/dwood/dev/clipspy/clips_source'
ld -G clips_source/*.o -o clips_source/libclips.so
python setup.py build_ext
Traceback (most recent call last):
  File "setup.py", line 32, in <module>
    from setuptools import find_packages, setup
ImportError: No module named setuptools
make: *** [Makefile:35: clipspy] Error 1
`

running python setup.py build_ext outside of sudo works without issue.

**Comment by noxdafox** [comment]
Hello,

`clipspy` master branch is under work at the moment in order to release `6.40` compatible APIs. Hence, logic is expected to be temporarily unstable.

I would recommend you to install it from `pip` (6.30) for the time being.

**Comment by david9100** [comment]
I figured that but thought I'd pass along the feedback.  BTW, I use clips 6.4 very heavily via the c api and am in the process of spinning up a python rules engine using your lib.  Would be happy to provide testing feedback if interested.

**Comment by noxdafox** [comment]
Now `clipspy` 1.0.0 is shipped as binary wheel which should be compatible with most of the recent Linux versions.

It should be enough to install it via `pip`.

Thanks for reporting this.

**Comment by david9100** [comment]
I did a bit more digging since I last posted and the problem seems to stem from line 57 of your make file in which you run the setup script.  The $(PYTHON) variable which resolves to **python** is only available when you are running under virtualenv.  As the problem is invoked when running "**sudo make install**", the root user is not operating under the virtual environment and the command python is not available.  The default command is **python3**.  I simply resolved the issue by changing the declaration on line 1 of your makefile.  Hope that helps.
